catroid/res/values/styles.xml

```diff
@@ -443,6 +443,23 @@
         <item name="android:background" >@drawable/brick_orange_3h</item >
     </style >
 
+    <style name="BrickContainer.Event" ></style >
+
+    <style name="BrickContainer.Event.Small" >
+        <item name="android:minHeight" >@dimen/brick_height_small</item >
+        <item name="android:background" >@drawable/brick_brown_1h_when_9</item >
+    </style >
+
+    <style name="BrickContainer.Event.MediumWhen9" >
```

Comment:
The 9 stands for 9 Patch Graphik, no need to include it in the style name, this is just confusing ;D